### ISSUE TITLE
Fix commit history

### ISSUE THREAD

**Issue by espes** [Echelon9/cxbx-shogun#18]
I went to some trouble piecing together the various historical repositories of Cxbx. It'd be cool if you rebase onto https://github.com/xqemu/Cxbx when you get a chance.

**Comment by Echelon9** [comment]
Yes, there's a degree of splintered forks going on at the moment. I'll see what I can do sensibly.
